string request_message
---
string response_message